iosApp/.swiftlint.yml

```diff
@@ -116,5 +115,5 @@ custom_rules:
   multiline_arguments_one_per_line:
     name: 'Multiline Arguments One Per Line'
     message: 'Arguments should be either on the same line, or one per line.'
-    regex: '[^\n\r]\(\n([^\(])*([^\n\r],([^\n\r])*[\w]+)([^\)])*\n(\s)*\)'
+    regex: '[^\n\r]\(\n([^\(<])*([^\n\r],([^\n\r])*[\w]+)([^\)])*\n(\s)*\)'
```

Comment:
@minhnimble There is a bug that it will trigger the lint warning for this line
```
    func find(
         _ elementKey: KeyPath<XCUIApplication, XCUIElementQuery>,
         with identifier: Identifier
     ) -> XCUIElement {
         return application[keyPath: elementKey][identifier.rawValue]
     }
```
because inside `<XCUIApplication, XCUIElementQuery>` there is a `,`

Reply:
noted, thanks for the insights 🙏